src/includes/getting-started-install/native.mdx

```diff
@@ -6,9 +6,9 @@ For example, `CMake` can be used like this (on macOS):
 
 ```shell
 # configure the cmake build into the `build` directory, with crashpad (on macOS)
-cmake -B build -D SENTRY_BACKEND=crashpad
+cmake -B build -D SENTRY_BACKEND=crashpad --config RelWithDebInfo 
 # build the project
-cmake --build build --parallel
+cmake --build build --config RelWithDebInfo --parallel
```

Comment:
can you also re-generate the file tree? now that we install debug symbols properly? (or to check if we actually do)

Reply:
Are you asking if the pdb gets copied over on install?

Reply:
bump?